src/tools/bin/egetopt

```diff
@@ -149,4 +163,4 @@ while true ; do
     shift
 done    
 
-getopt $getOptArgs -- $transformedArgs
+echo "getopt $getOptArgs -- $transformedArgs" | bash
```

Comment:
The first line is only absolute  because it is [not allowed](http://en.wikipedia.org/wiki/Shebang_%28Unix%29) to use the interpreter name only.